### PR TITLE
Lesson 1.3 technical review

### PR DESCRIPTION
**Pull request by andrewk**
Corrects and clarifies semver and NPM

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ manager for node. Although, npm does *not* stand for "node package manager".
 I won't go into the details of that, but if you'd like to, you can read all
 about it [here](https://docs.npmjs.com/misc/faq#if-npm-is-an-acronym-why-is-it-never-capitalized).
 
-npm makes it really easy to install, and maintain specific versions of
```
Comment (andrewk) [inline readme.md:8]: Over emphasising the simplicity of a technical area the reader is studying can have unintentional negative impact. 
`npm` can be far from easy.

File: readme.md [hunk 2]
```diff
@@ -5,14 +5,14 @@ manager for node. Although, npm does *not* stand for "node package manager".
...
+npm's purpose is to install and maintain specific versions of
 JavaScript packages without the old copy and paste that we used to have to do
 back in the "old days".
 
 ## Dependencies
 
-Application requirements are defined in a file that typically lives in the
```
Comment (andrewk) [inline readme.md:14]: No need for inferring optional further configuration complications.

File: readme.md [hunk 3]
```diff
@@ -57,19 +57,17 @@ ancillary information that is needed if we are going to push our package up to n
 
 ## Semantic Versioning
 
-npm uses [semantic versioning](http://semver.org/) in all of their modules
```
Comment (andrewk) [inline readme.md:60]: > in all of their modules

not _their_ modules

File: readme.md (hunk 1 above)
Comment (dphaener) [inline readme.md:8]: Agreed. :+1:

**Pull request merged**